scripts/check-tags.ts

```diff
@@ -38,16 +49,24 @@ const checkProjects = async () => {
   const projects = globSync('{apps,libs}/**/project.json', {
     ignore: '**/node_modules/**',
   })
-  let hasError = false
+  let invalidProjects = []
   for (const project of projects) {
-    hasError = hasError || (await checkTags(project))
+    // console.log("Checking project", project)
+    // Old
+    // hasError = hasError || (await checkTags(project))
+
+    // New, and faster
+    if (!(await hasValidTags(project).catch(() => false))) {
+      invalidProjects.push(project)
+    }
   }
 
-  if (hasError) {
-    console.log(chalk.red('Found errors'))
+  if (invalidProjects.length > 0) {
+    console.log(chalk.red('Found errors in project files'))
     console.log(
-      'All projects should have a configured NX tags which controls which project can import what. For more information see: https://docs.devland.is/repository/nx-tags',
+      'All projects should have a configured NX tags which controls which project can import what. For more information see: https://docs.devland.is/repository/NX-tags',
     )
+    console.log('Invalid projects:', invalidProjects)
```

Comment:
Can be formatted better?

Reply:
This should make javascript output in the best manner possible, multilined colour when running locally, single-line no colour when running without TTY in CI.

Reply:
Meh, was just thinking more NX style terminal output:

```
Invalid projects:

- project1
- project2
- project3
```

Instead of JSON style output:

```
Invalid projects: [
  "project1",
  "project2",
  "project3"
]
```